techniques/system/common/1.0/promises.st

```diff
@@ -190,7 +190,7 @@ bundle agent check_disable_agent
 
       # Only define this class when we're ready to die - this is a special class name in "abortclasses"
       "should_not_continue"
-        expression => "should_disable_agent",
+        expression => "should_disable_agent.!ignore_disable_agent",
```

Comment:
We don't have a rudder agent inventory --force and an associated class, maybe we should have a ignore_disable_inventory class instead